wms/src/main/java/bg/tuvarna/sit/wms/entities/Address.java

```diff
@@ -0,0 +1,35 @@
+package bg.tuvarna.sit.wms.entities;
+
+import bg.tuvarna.sit.wms.entities.base.BaseEntity;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
```

Comment:
It is better not to use wildcards (*) in the import statements.
You can use Alt + Enter to fix this.